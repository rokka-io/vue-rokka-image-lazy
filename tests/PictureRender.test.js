import { mount } from '@vue/test-utils'
import Picture from './PictureRender.vue'

let wrapper = mount(Picture)

describe('RokkaPicture rendering', () => {
  test('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
