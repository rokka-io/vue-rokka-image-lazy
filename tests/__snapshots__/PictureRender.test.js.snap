// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RokkaPicture rendering renders correctly 1`] = `
<div
  class="rokka-picture"
>
  <picture>
    <source
      class="rokka--attr-data-srcset"
      data-cache="https://testorg.rokka.io/dynamic/resize-width-100-height-1000--grayscale/o-af-1/1234/image.jpg"
      data-srcset="https://testorg.rokka.io/dynamic/resize-width-100-height-1000--grayscale/o-af-1/1234/image.jpg 1x, https://testorg.rokka.io/dynamic/resize-width-100-height-1000--grayscale/o-af-1-dpr-2/1234/image.jpg 2x"
      media="(max-width: 1000px)"
    />
     
    <source
      class="rokka--attr-data-srcset"
      data-cache="https://testorg.rokka.io/dynamic/resize-width-100-height-1000/o-af-1/1234/image.jpg"
      data-srcset="https://testorg.rokka.io/dynamic/resize-width-100-height-1000/o-af-1/1234/image.jpg 1x, https://testorg.rokka.io/dynamic/resize-width-100-height-1000/o-af-1-dpr-2/1234/image.jpg 2x"
      media="all"
    />
     
    <img
      alt="foo"
      class="rokka--attr-data-src"
      data-cache="https://testorg.rokka.io/dynamic/resize-width-100-height-1000/o-af-1/1234/image.jpg"
      data-src="https://testorg.rokka.io/dynamic/resize-width-100-height-1000/o-af-1/1234/image.jpg"
      data-srcset="https://testorg.rokka.io/dynamic/resize-width-100-height-1000/o-af-1/1234/image.jpg 1x, https://testorg.rokka.io/dynamic/resize-width-100-height-1000/o-af-1-dpr-2/1234/image.jpg 2x"
      src="/loader.svg"
      title="image title"
    />
  </picture>
</div>
`;
